app/src/test/kotlin/no/nav/arbeidsgiver/notifikasjon/kafka_backup/BackupRepositoryTest.kt

```diff
@@ -135,11 +136,11 @@ fun record(
         offset,
         0L,
         TimestampType.NO_TIMESTAMP_TYPE,
-        0L,
         keyBytes.size,
         valueBytes?.size ?: 0,
         keyBytes,
         valueBytes,
         headers,
+        Optional.of(0)
```

Comment:
fiks for compiler warning om deprecated constructor som er markert for fjerning i kafka 4.x